resources/abl-src/dynrun.p

```diff
@@ -42,28 +42,41 @@ ASSIGN configJson = CAST(jsonParser:ParseFile(SESSION:PARAMETER), JsonObject).
 LOG-MANAGER:WRITE-MESSAGE(SUBSTITUTE("JSON Config file: &1", SESSION:PARAMETER)).
 OS-DELETE VALUE(SESSION:PARAMETER).
 
-// DB connections + aliases
-ASSIGN dbEntries = configJson:GetJsonArray("databases").
-DO zz = 1 TO dbEntries:Length:
-  ASSIGN dbEntry = dbEntries:GetJsonObject(zz).
-  IF (dbEntry:has("name") AND dbEntry:has("connect")) THEN DO:
-    LOG-MANAGER:WRITE-MESSAGE(SUBSTITUTE("Connecting to DB '&1': '&2'", dbEntry:GetCharacter("name"), dbEntry:GetCharacter("connect"))).
-    CONNECT VALUE(dbEntry:GetCharacter("connect")) NO-ERROR.
-    IF ERROR-STATUS:ERROR THEN DO:
-      IF (ERROR-STATUS:NUM-MESSAGES > 1) OR (ERROR-STATUS:GET-NUMBER(1) NE 1552) THEN DO:
-        LOG-MANAGER:WRITE-MESSAGE(SUBSTITUTE("Unable to connect to '&1'" , dbEntry:GetCharacter("name"))).
-        DO i = 1 TO ERROR-STATUS:NUM-MESSAGES:
-          LOG-MANAGER:WRITE-MESSAGE(ERROR-STATUS:GET-MESSAGE(i)).
+//DB connections + aliases
+DO ON ERROR UNDO, LEAVE:
```

Comment:
Wouldn't configJson:has("databases") be better ?

Reply:
Most certainly better!  Updated.